website/docusaurus.config.js

```diff
@@ -31,19 +32,7 @@ if (!PRERELEASE) {
   }
 }
 
-var ALGOLIA_API_KEY;
-if (!process.env.ALGOLIA_API_KEY) {
-  ALGOLIA_API_KEY = '0e9665cbb272719dddc6e7113b4131a5';
-} else {
-  ALGOLIA_API_KEY = process.env.ALGOLIA_API_KEY;
-}
-
-var ALGOLIA_INDEX_NAME;
-if (!process.env.ALGOLIA_INDEX_NAME) {
-  ALGOLIA_INDEX_NAME = 'dbt';
-} else {
-  ALGOLIA_INDEX_NAME = process.env.ALGOLIA_INDEX_NAME;
-}
+let { ALGOLIA_APP_ID, ALGOLIA_API_KEY, ALGOLIA_INDEX_NAME } = process.env;
```

Comment:
this looks great! Just wondering - do these env vars get passed in from Netlify in the staging/prod envs?

Reply:
> this looks great! Just wondering - do these env vars get passed in from Netlify in the staging/prod envs?

Yep! It pulls them in with the same `process.env`. I kept the debug console.log in place for the ALGOLIA_INDEX_NAME to verify this within the Netlify build logs. On our last preview build log it has `9:59:53 AM: DEBUG: ALGOLIA_INDEX_NAME =  dbt`. This would be undefined if it failed to grab the Netlify env variable.

Reply:
> Was about to write up an issue, but I think this will resolve it! 👍

@kwigley The build just completed, looks like search is now working on my end. Can you verify you're no longer seeing the errors on your end?

Reply:
@JKarlavige no errors, looks great! 🎉